### DISCUSSION TITLE
Validation objects could be more useful

### DISCUSSION THREAD

**Discussion by dominictobias-bullish** [elysiajs/elysia#675] [Ideas]
At the moment if I don't return an expected `response` that I've typed, Elysia has the following error object:

```
{
"type": "response",
"validator": {},
"value": {},
"code": "VALIDATION",
"status": 422
}
```

For large objects it takes me ages to manually check every field of often an array of objects from an external server to see what I've typed too strictly. As far as I can tell this error object has nothing useful like "Expected 'lastPrice' to be a number". It just has the validation schema and the values.

It would be great if there was also a message like `messages: ["'subtitle' is required",  "'page_number' must be numeric"]`. This could also form better UI messages.

**Comment by cannap** [comment]
as far i know you have to reMap them onError anyway in production validations errors are empty

**Reply by frhnfrq** [reply]
@cannap remapping it in onError is unfortunately not being reflected in the Eden client, it still shows the default validation error type
